gui-apps/river-tag-overlay/river-tag-overlay-1.0.0.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit git-r3
+
+DESCRIPTION="A pop-up showing tag status [in the river window manager]"
+HOMEPAGE="https://git.sr.ht/~leon_plickat/river-tag-overlay"
+#SRC_URI="https://git.sr.ht/~leon_plickat/river-tag-overlay"
+EGIT_REPO_URI="https://git.sr.ht/~leon_plickat/river-tag-overlay"
+
+LICENSE="GPLv3"
+SLOT="0"
+KEYWORDS="~arm64 ~amd64"
```

Comment:
Please only add `~amd64`, I'm not going to be able to check on `~arm64`, plus you can always enable `**` for an overlay. Take a look at the readme for more details.

Reply:
I "developed" and tested the ebuild on ARM64. Does that suffice, or should I still remove the tag?

Reply:
Sure, but then you are the person who has to maintain it :smile_cat: . 
So please also add a metadata.xml file